src/components/key_pad_menu/key_pad_menu_item.tsx

```diff
@@ -178,6 +179,7 @@ export const EuiKeyPadMenuItem: FunctionComponent<EuiKeyPadMenuItemProps> = ({
     },
     className
   );
+  const betaItemLabel = useEuiI18n('euiKeyPadMenuItem.betaLabel', 'Beta item.');
```

Comment:
Sorry, I could've provided more guidance on the i18n util; including the `label` variable here will make the whole label localizable, instead of forcing the order.

```suggestion
  const betaItemLabel = useEuiI18n(
    'euiKeyPadMenuItem.betaLabel',
    '{label}. Beta item.',
    { label }
  );
```

and then,

```
aria-label={betaItemLabel}
```

Reply:
I'll grab this on Tuesday morning right away!